scripts/ios/helper.js

```diff
@@ -82,6 +82,11 @@ module.exports = {
 
         // Generate a unique ID for our new build phase.
         var id = xcodeProject.generateUuid();
+
+        if (xcodeProject.hash.project.objects.PBXShellScriptBuildPhase) {
+            xcodeProject.hash.project.objects.PBXShellScriptBuildPhase = {};
```

Comment:
you want to reset it if it exists or create it if it doesn't exist? @rubnogueira